### DISCUSSION TITLE
https://unifi.boostchicken.io/ Down?

### DISCUSSION THREAD

**Discussion by brandonhon** [unifi-utilities/unifi-common#473] [Q&A]
I'm not able to get the the site or download zip files.

I was also wondering if this is working on UniFi OS SE v3.0.13?

**Comment by chriswiggins** [comment]
Experiencing the same issue

**Reply by brandonhon** [reply]
I was able to eventually download the file from unifi.boostchicken.io. still wondering if this works on UniFi OS SE v3.0.13.

Try this. the -s make curl silent so you do not see the progress. Be patient and it will eventually start.
`curl -Lfo udmse-podman-install.zip https://unifi.boostchicken.io/udmse-podman-install.zip`
